Makefile.features

```diff
@@ -0,0 +1,8 @@
+# import list of provided features
+-include $(RIOTBOARD)/$(BOARD)/Makefile.features
+-include $(RIOTCPU)/$(CPU)/Makefile.features
+
+DEFAULT_FEATURES += periph_pm
+
+# add available default features to required list
+FEATURES_REQUIRED += $(filter-out $(DISABLE_FEATURES), $(filter $(FEATURES_PROVIDED), $(DEFAULT_FEATURES)))
```

Comment:
I think in your last statement we have the problem: `FEATURES_REQUIRED` should be `(DEFAULT_FEATURES & FEATURES_PROVIDED) - DISABLED_FEATURES`?!